src/main/java/games/strategy/engine/data/TripleaHandler.java

```diff
@@ -0,0 +1,475 @@
+package games.strategy.engine.data;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Deque;
+import java.util.LinkedList;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.SAXParseException;
+import org.xml.sax.helpers.AttributesImpl;
+import org.xml.sax.helpers.DefaultHandler;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+
+/**
+ * Abstract common implementation of a DefaultHandler.
+ */
+public abstract class TripleaHandler extends DefaultHandler {
```

Comment:
TripleAHandler is a checkstyle violation, because capital A is being followed by a capital S.
Agreed, TripleAHandler is probvably not a good name, but I'd rather go for something like `TripleaSaxHandler`